extensions/rest/src/main/java/org/radargun/traits/RESTOperations.java

```diff
@@ -0,0 +1,22 @@
+package org.radargun.traits;
+
+import java.util.List;
+import javax.ws.rs.core.Cookie;
+import org.radargun.Operation;
+import org.radargun.WrappedHttpResponse;
+
+/**
+ * @author Martin Gencur
+ */
+@Trait(doc = "Http operations.")
+public interface RESTOperations {
+   String TRAIT = RESTOperations.class.getSimpleName();
+
+   Operation GET = Operation.register(TRAIT + ".Get");
+
+   RESTOperationInvoker getRESTInvoker();
+
+   interface RESTOperationInvoker {
```

Comment:
I will move the get operation directly to RESTOperations, the invoker is probably not needed. Regarding the trait and URL, currently there's only one implementation of RESTOperations and that is tightly couple with RESTEasyService which defines the URL to be used. I'd have to immediately ignore this new String parameter in the single implementation because I want to define the URL directly in the configuration file. We can add this later if there's a need to request different URLs during a single stage.